SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -0,0 +1,70 @@
+# Get started: 3 - Set up the Unreal GDK module and Starter Project
+
+Follow the steps below to:
+
+* Clone the Starter Project repository.
+* Clone the Unreal GDK into the `Plugins` folder of your game.
+* Build the Unreal GDK module dependencies which the Starter Project needs so it can work with the GDK.
+* Run the Starter Project on SpatialOS
+
+### Cloning
+
+1. Clone the [Unreal GDK Starter Project](https://github.com/spatialos/UnrealGDKStarterProject/) repository by running either:
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDKStarterProject.git`
+    * (SSH) `git clone git@github.com:spatialos/UnrealGDKStarterProject.git`
+1. Navigate into `<StarterProjectRepoRoot>/Game/Plugins`. You may need to create the Plugins folder if you don't have any plugins installed yet.
+1.  In a Git Bash terminal window, clone the [Unreal GDK](https://github.com/spatialos/UnrealGDK) repository by running either:
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDK.git`
+    * (SSH) `git clone git@github.com:spatialos/UnrealGDK.git`
+
+**Note** You need to ensure that the root folder of the Unreal GDK repo is called UnrealGDK (`<StarterProjectRepoRoot>/Game/Plugins/UnrealGDK/...`)
+
+[//]: # (TODO: This whole section below deserves some screenshots to show users what to expect. This was done well on the Unity onobarding docs. Example: https://docs.improbable.io/unity/alpha/content/get-started/get-playing)	
+
+### Building
+
+> If you want to port an existing Unreal project to the Unreal GDK, follow [this guide]({{urlRoot}}/content/porting-unreal-project-to-gdk) instead of continuing to follow these steps.
+
+Build the Unreal GDK module dependencies which the Starter Project needs to work with the GDK and add the Unreal GDK to the Starter Project.
```

Comment:
I remember this is a bit of an odd one - maybe something like:

"In this step, you're going to build the Unreal GDK module dependencies, which the Starter Project needs to work with the GDK. You are then going to add the Unreal GDK to the Starter Project."

Reply:
agreed